sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/ServiceItemLease.java

```diff
@@ -63,7 +63,8 @@ public ServiceItemLease withId(String id) {
         return this;
     }
 
-    @JsonIgnore
+//    @JsonIgnore
+    @JsonProperty("_etag")
```

Comment:
when a class is annotated with jackson annotation, jackson uses reflection for serialization/deserialization to json.

reflection is expensive and that being said, If there are pre existing classes in the code which needs to be serialized/deserialized to json we should ideally use a jackson serializer rather than annotating the class with jackson annotation.

(not necessarily in the scope of this PR, but something to be addressed outside of this PR).

Reply:
n this case the "_etag" field was not passed through the serialization and that resulted in "IF MATCH" condition to fail when updating the lease document.

Reply:
The ask is to add something similar to this:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/routing/PartitionKeyInternal.java#L35-L36

The internal types should not rely on reflection for serialization/deserialization.

Reply:
Fixed. We only need a serializer here, the de-serialization is done at the Document/CosmosItemProperties level and that instance it is later used to create a Lease instance.

Reply:
If you have a serializer you need a deserializer too. 

Document/CosmosItemProperties should call to the custom deserializer for deserialization. no?